statics/js/components/capture-form.js

```diff
@@ -39,7 +39,13 @@ Vue.component('capture-form', {
         </div>\
         <div class="form-group">\
           <label for="capture-bpf">BPF filter</label>\
-          <input id="capture-bpf" type="text" class="form-control input-sm" v-model="bpf" />\
+          <div class="input">\
+            <input list="capture-bpf-favorites" placeholder="e.g. ip host skydive.rdocloud"\
```

Comment:
I would prefer to use skydive.rdocloud as example. I would rather use "port 80 and port 443"

Reply:
done